sdks/python/apache_beam/runners/worker/sdk_worker.py

```diff
@@ -607,12 +607,18 @@ def _schedule_periodic_shutdown(self):
     # type: () -> None
     def shutdown_inactive_bundle_processors():
       # type: () -> None
+      inactive_descriptor_ids = []
+      cur_time = time.time()
```

Comment:
nit: could calculate threshold here since it's same for all elements, think less line wraps